src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -283,6 +289,29 @@ private static bool MatchWithQueryString(string key, string[] subPatterns)
             return (double)numberOfMatchSymbols / numberOfAllSymbols > 0.8;
         }
 
+        /// <summary>
+        /// Check if any element in the given subset
+        /// matches any element in the tagDictionary.
+        /// </summary>
+        /// <param name="subset">The subset that will be used to match elements in tagDictionary.</param>
+        /// <returns></returns>
+        private static bool MatchWithSubset2(IGrouping<string, Tuple<V, double>> pair, IEnumerable<SearchElements.NodeSearchElement> subset)
```

Comment:
Is there a version1 of this function, if not, let's get rid of the `2` in function name?